public/src/components/epicTests/epicTestEditor.tsx

```diff
@@ -287,7 +287,7 @@ class EpicTestEditor extends React.Component<EpicTestEditorProps, EpicTestEditor
                 className={classes.select}
                 multiple
                 value={test.locations}
-                onChange={this.onLocationsChange}
+                onChange={() => this.onLocationsChange}
```

Comment:
this change means it will not actually call `this.onLocationsChange`

Reply:
I think I've fixed it now with latest commit.